### DISCUSSION TITLE
SVG to PNG/JPG conversion issue

### DISCUSSION THREAD

**Discussion by shivanshu-sigh-dev** [ImageMagick/ImageMagick#4470] [Help]
Hello Everyone!

I was trying to convert a SVG file into PNG/JPG using this tool. The command I was using is (magick.exe -density 400 test.svg -quality 75% test.png).

The conversion is working fine, even I am getting a nice picture quality as well but 

- There is a text in the SVG "speed" after conversion in the image it is printed as "sp eed"
- Some of the text are getting outside of the box with different color.

Could anyone help me please in resolving this issue.

Thank you!

**Comment by snibgo** [comment]
How can we help without seeing your SVG?

Please also say what version of IM, on what platform, and give the text output from "magick.exe -verbose test.svg NULL:"

**Reply by shivanshu-sigh-dev** [reply]
I am using the IM version 7 on Windows 10
Output from the command mentioned by you
"test.svg SVG 794x1123 794x1123+0+0 16-bit sRGB 213513B 0.234u 0:00.390
test.svg=> SVG 794x1123 794x1123+0+0 16-bit sRGB 213513B 0.000u 0:00.000
demo.png SVG 3307x4677 3307x4677+0+0 16-bit sRGB 213513B 1.922u 0:01.091"

![image](https://user-images.githubusercontent.com/67796452/141794660-89b1e1b0-48cb-4d4e-be61-e8e52ffee1b0.png)

Please have a look on the above image for the differences between origional and converted especially related to text.

**Reply by fmw42** [reply]
Please post your SVG file and show your full command line and add -verbose to it so we can see which SVG renderer you are using. Suggest you put Inkscape on your system so ImageMagick can render with it.
